avocado/plugins/distro.py

```diff
@@ -353,7 +353,7 @@ def configure(self, parser):
                                  parser=parser,
                                  long_arg='--distro-def-path')
 
-        type_choices = DISTRO_PKG_INFO_LOADERS.keys()
+        type_choices = tuple(DISTRO_PKG_INFO_LOADERS.keys())
```

Comment:
Any reason for using `tuple` instead of `list` here?

Reply:
It was just to follow the docs for `settings.register_option()`, but `list` should work fine as well from what I can remember.